install.sh

```diff
@@ -25,58 +26,80 @@ pip3 install pexpect
 pip3 install IKP3db
 pip install ikpdb
 
-echo "create folder structure"
+echo "###removing /etc/c9 if exists###"
+WORKING_DIR=/etc/c9
+if [ -d "$WORKING_DIR" ]; then rm -Rf $WORKING_DIR; fi
+
+echo "###create folder structure###"
 mkdir /etc/c9
 mkdir /etc/c9/sock
 mkdir /etc/c9/util
 
-echo "download utils"
+echo "###download utils###"
 wget https://raw.githubusercontent.com/kikito/md5.lua/master/md5.lua -O /etc/c9/util/md5.lua
 wget https://raw.githubusercontent.com/HuJK/Cloud9Hub/master/pip2su.py -O /etc/c9/util/pip2su.py
 wget https://raw.githubusercontent.com/HuJK/Cloud9-hub/master/create_login.py -O /etc/c9/util/create_login.py
 
-echo "install c9 sdk by offical installitaion script"
+echo "###install c9 sdk by offical installitaion script###"
 cd /etc/c9
 HOME=/etc/c9
 git clone https://github.com/c9/core sdk
 cd sdk
 ./scripts/install-sdk.sh
 
-echo "set permission"
+echo "###set permission###"
 chmod -R 755 /etc/c9/sdk
 chmod -R 755 /etc/c9/.c9
 chmod -R 755 /etc/c9/util
 chmod -R 773 /etc/c9/sock
 HOME=/root
+
 # Install npm back
 apt-get install -y npm
 
-echo "add nginx to shadow to make pam_module work"
+echo "###checking username if exists###"
```

Comment:
This account should be created by nginx, not by us.
But which name (nginx  or www-data) will be created is depends on nginx version. 

So I tried to add both to shadow group, but only one will success.